modules/rosa-cluster-autoscaler-ui-settings.adoc

```diff
@@ -81,32 +108,32 @@ The tables explain all the configurable UI settings when using cluster autoscali
 |Type or Range
 |Default
 
-|`enable`
+|`enabled`
```

Comment:
@aldiazRH There was a bug in the UI related to this field, and I realised that the label for it should be aligned with the others, meaning it will be changed to `scale-down-enabled`.

Apologies for the constant changes